os-eap-launch/added/launch/security-domains.sh

```diff
@@ -47,23 +47,11 @@ function configure_security_domains() {
       fi
       security_domain="$security_domain >"
 
-      local options=$(compgen -v | grep -s "${domain_prefix}_MODULE_OPTION_")
+      local options=$(compgen -v | grep -sE "${domain_prefix}_MODULE_OPTION_[a-zA-Z]*(_[a-zA-Z]*)*_NAME")
```

Comment:
Would this be a little easier if it was _NAME_<id> as opposed to _<id>_NAME?  Generally, I think this is a more common pattern.